README.md

```diff
@@ -273,6 +273,97 @@ manually).  One way to do this, on Linux, is to ensure that the
 
 (or whatever directory `libmlpack.so` is installed in.)
 
+#### Building with Visual Studio 2019 
```

Comment:
Hmm, do you think that we should make this a standalone documentation page?  It's probably good to try and keep the readme short.  Maybe we can add this to `doc/guide/build_windows.hpp` or another file in that directory?  Let me know what you think. :+1:

Reply:
I can certainly do that, I'll have a look at that location. But agreed, this section would be quite chunky on the `README.md`, so perhaps just a link to the new document would improve things.

Reply:
When I get some more time today, I'll move those over. Thanks for the review @rcurtin .

Reply:
There are some links already to the Windows build directions:

```
There are some useful pages to consult in addition to this section:

   * Building mlpack From Source
   * Building mlpack From Source on Windows
```

If you like, we can use that second link, or perhaps we could add a third link specifically for this VS2019 integration (either linking to a separate page than the Windows build or to an anchor on the Windows build page).  Your call on what the best way is. :+1:

Reply:
@rcurtin I read those instructions earlier today. The instructions I provided above could be considered an alternative to the ones currently in the docs. Instead of generating the VS project with CMake, we're using VS's own CMake tools configure & generate the project.

I could incorporate the instructions above into the current docs, thus having the following sections for "opening" the project with VS:

- Generating a Visual Studio solution with CMake: this is the current instructions.
- Using VS's CMake Integration: directly work with Visual Studio's CMake integration to build the project (the instructions I added).

Also, the added OpenBLAS comments could be incorporated into the current instructions, then having the sections:

- Getting prebuilt OpenBLAS with Nuget (VS's package manager, currently in the docs).
- Building OpenBLAS with Visual Studio from source (instructions I added).

I think it's definitely best to move the changes regarded in this MR to `doc/guide/windows_build.hpp`, but I will need to incorporate these as mentioned in the points above, so it all reads smoothly :) 

I'll make these changes at some point tomorrow! Thanks again for reviewing.

Reply:
@rcurtin please don't resolve this discussion until I make these changes :)

Reply:
Sounds good. :+1: I try to only resolve conversations that have been fully handled---if I resolved any too soon, feel free to unresolve them. :)

Reply:
@rcurtin I have incorporated these changes to the `doc/guide/build_windows.hpp` file. I haven't yet removed the ones in `README.md` for tracking purposes, but when we establish that the docs changes are satisfactory, I'll rebase and remove them :)

Reply:
@rcurtin let me know if any styling changes are needed as well, I might have missed something.

Reply:
Looks great!  Thank you for taking the time to do that.  It looks like it is also in the README so maybe we should revert that change?

Reply:
Have reverted and cleaned up the commit history. Thanks for reviewing @rcurtin .